stable/keycloak/README.md

```diff
@@ -230,6 +231,25 @@ Alternatively, the file could be added to a custom image (set in `keycloak.image
 
 After startup the web admin console for the realm should be available on the path /auth/admin/\<realm name>/console/.
 
+### Using kcadm cli with postStartScript
+
+Keycloak can be configured using the kcadm cli. For example, SSL for master realm is enabled by default but can be disabled with:
```

Comment:
I don't think this is a good example, since deactivating SSL is something I'd not generally recommend? Using secrets in plain-text is not a good idea either. Another general problem is that this is run on all replicas and on every restart, so the user needs to be careful what they do. Scripts have to be idempotent.

The question really is if this has to be part of the Helm chart. I think you'd be better off running a separate script once after installing the chart against a single replica that does this:

```
kubectl exec keycloak-0 ./keycloak/bin/kcadm.sh ...
```

You could also copy a file into the container:

```
kubectl cp ./activiti-client.json keycloak-0:/activiti-client.json
kubectl exec keycloak-0 ./keycloak/bin/kcadm.sh update clients/<ID> -f /activiti-client.json ...
```

Reply:
Yeah I hadn't considered these problems.

What I actually is to package the keycloak chart inside an activiti chart using a provided realm and allow users to deploy it as they like in their own environments. That means they'll have to put their environment's url in for the redirectUris.

What I could do instead is use the preStartScript to do a sed and replace the provided realm's Uris with those provided by the user.

Reply:
I'd love it if the user of the activiti chart didn't have to put their url in at all. I previously tried using a * for these allowed urls but keycloak didn't allow that (presumably for security reasons). 

I've noticed other projects also ask a user to configure a uri for keycloak. (One being jhipster - see keycloak section of https://www.jhipster.tech/docker-compose/ Maybe I could ask @mraible about that.)

Another example of this where this chart is getting packaged in another chart is https://github.com/Alfresco/alfresco-identity-service/blob/master/helm/alfresco-identity-service/values.yaml.

Seems like others are also going to want to package this chart inside their own. Perhaps what I should do is only provide a script for updating the urls as that is an idempotent operation and add a short readme section on including this chart as a subchart. If we go that way then probably best to close this issue and create a new one.